homeassistant/components/zwave_js/strings.json

```diff
@@ -14,7 +14,7 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "config_entry_not_loaded": "The Z-Wave configuration entry is not loaded. Please try again when the configuration entry is loaded.",
       "different_device": "The connected USB device is not the same as previously configured for this config entry. Please instead create a new config entry for the new device.",
-      "discovery_requires_supervisor": "Discovery requires the supervisor.",
+      "discovery_requires_supervisor": "Discovery requires the Z-Wave Supervisor add-on.",
```

Comment:
This is not correct, this refers to a functionality of the supervisor, not the add-on.